pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,12 +54,13 @@ type Response struct {
 
 type Client struct {
 	*client.Client
-	Request  Request
-	Response Response
-	Error    bool
-	Created  bool
-	server   *httptest.Server
-	handler  *utiltesting.FakeHandler
+	Clientset *release_1_1.Clientset
```

Comment:
Thanks for the suggestion. I'm reluctant to rename it now because I'm afraid we'll change our mind later and wasting the effort :)